File: hackerrank/30 Days of Code/Day 2/arithmetic.clj


; Complete the solve function below.
(defn solve [meal_cost tip_percent tax_percent]
    (let [cost (* meal_cost (+ 1 (/ (+ tip_percent tax_percent) 100)))
          total_cost (Math/round cost)]
        (println (str "The total meal cost is " total_cost " dollars.")))
)

(def meal_cost (Double/parseDouble (clojure.string/trim (read-line))))

(def tip_percent (Integer/parseInt (clojure.string/trim (read-line))))

(def tax_percent (Integer/parseInt (clojure.string/trim (read-line))))

(solve meal_cost tip_percent tax_percent)

File: hackerrank/30 Days of Code/Day 2/arithmetic.py
import math
import os
import random
import re
import sys

# Complete the solve function below.


def solve(meal_cost, tip_percent, tax_percent):
    cost = meal_cost * (1 + (tip_percent + tax_percent) / 100.0)

    total_cost = round(cost)
    # print("The total meal cost is " + str(total_cost) + " dollars.")
    print("The total meal cost is {} dollars.".format(total_cost))

if __name__ == '__main__':
    meal_cost = float(input())

    tip_percent = int(input())

    tax_percent = int(input())

    solve(meal_cost, tip_percent, tax_percent)
